components/brave_rewards/browser/rewards_service_factory.cc

```diff
@@ -9,6 +9,8 @@
 #include "chrome/browser/profiles/incognito_helpers.h"
 #include "chrome/browser/profiles/profile.h"
 #include "components/keyed_service/content/browser_context_dependency_manager.h"
+#include "extensions/browser/event_router_factory.h"
```

Comment:
this should also be behind the buildflag, but not the buildflag header itself, of course